cluster/saltbase/salt/etcd/etcd.manifest

```diff
@@ -0,0 +1,45 @@
+{% if grains.etcd_servers is defined -%}
+  {% set etcd_server = "" + grains.etcd_servers -%}
+{% else -%}
+  {% set ips = salt['mine.get']('roles:kubernetes-master', 'network.ip_addrs', 'grain').values() -%}
+  {% set etcd_server = "" + ips[0][0] -%}
+{% endif -%}
+
+version: v1beta2
+id: etcd-server
+containers:
+  - name: etcd-container
+    image: kubernetes/etcd:2.0.1
+    command: [
+              "/usr/local/bin/etcd",
+	      "--addr",
+	      "{{etcd_server}}:4001",
+	      "--bind-addr=0.0.0.0:4001",
+	      "--data-dir",
+	      "/var/etcd",
+	      "--initial-advertise-peer-urls",
+	      "http://kubernetes-master:2380",
+	      "--name",
+	      "kubernetes-master",
+	      "--initial-cluster",
+	      "kubernetes-master=http://kubernetes-master:2380",
+            ]
+    ports:
+      - name: http2
+        containerPort: 7001
```

Comment:
http1,2,3 are out of order.

Can you give these better names? 

In the etcd bug they pointed out that they don't use 4001/7001 any more - why are we?